libc/config/windows/headers.txt

```diff
@@ -0,0 +1,31 @@
+set(TARGET_PUBLIC_HEADERS
+    libc.include.assert
+    libc.include.ctype
+    libc.include.dirent
+    libc.include.errno
+    libc.include.features
+    libc.include.fenv
+    libc.include.float
+    libc.include.inttypes
+    libc.include.limits
+    libc.include.locale
+    libc.include.malloc
+    libc.include.math
+    libc.include.sched
+    libc.include.search
+    libc.include.setjmp
+    libc.include.signal
+    libc.include.stdbit
+    libc.include.stdckdint
+    libc.include.stdfix
+    libc.include.stdint
+    libc.include.stdio
+    libc.include.stdlib
+    libc.include.string
+    libc.include.strings
+    libc.include.threads
+    libc.include.time
+    libc.include.uchar
+    libc.include.unistd
+    libc.include.wchar
```

Comment:
I'd recommend limiting this to only headers which have functions enabled on windows:
```suggestion
    libc.include.ctype
    libc.include.string
    libc.include.inttypes
    libc.include.stdlib
    libc.include.errno
    libc.include.fenv
    libc.include.math
```